truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -384,19 +384,21 @@ def prepare_trtllm_encoder_build_dir(self, build_dir: Path):
         # to 32 even if the engine.rank0 allows for higher batch_size
         runtime_max_batch_size = min(config.trt_llm.build.max_batch_size, 32)
         port = 7997
-        start_command = (
-            f"python-truss-download && text-embeddings-router "
-            f"--port {port} "
-            f"--max-batch-requests {runtime_max_batch_size} "
-            # how many sentences can be in a single json payload.
-            # limited default to improve request based autoscaling.
-            f"--max-client-batch-size {ENCODER_TRTLLM_CLIENT_BATCH_SIZE} "
-            # how many concurrent requests can be handled by the server until 429 is returned.
-            # limited by https://docs.baseten.co/performance/concurrency#concurrency-target
-            # 16384 is a safe max value for the server
-            f"--max-concurrent-requests 16384"
-            # downloaded model path by `python-truss-download` cmd
-            "--model-id /app/data/tokenization"
+        start_command = " ".join(
+            [
+                "python-truss-download && text-embeddings-router",
+                f"--port {port}",
+                f"--max-batch-requests {runtime_max_batch_size}",
+                # how many sentences can be in a single json payload.
+                # limited default to improve request based autoscaling.
+                f"--max-client-batch-size {ENCODER_TRTLLM_CLIENT_BATCH_SIZE}",
+                # how many concurrent requests can be handled by the server until 429 is returned.
+                # limited by https://docs.baseten.co/performance/concurrency#concurrency-target
+                # 2048 is a safe max value for the server
+                "--max-concurrent-requests 2048",
```

Comment:
nit: can we put this magic number in a constant somewhere?

Reply:
Okay, done!